pkg/vault/contracts/Router.sol

```diff
@@ -449,7 +449,7 @@ contract Router is IRouter, RouterCommon, ReentrancyGuardTransient {
     function removeLiquidityRecovery(
         address pool,
         uint256 exactBptAmountIn
-    ) external returns (uint256[] memory amountsOut) {
+    ) external payable returns (uint256[] memory amountsOut) {
```

Comment:
Not a problem specific to this PR, but if I just call this function with ETH it will get stuck in the router until somebody else calls a function that returns it. Doesn't sound like the best design idea.

Reply:
Calling a remove with ETH by itself isn't something that should ever happen - but this effect is certainly unintended. I don't think it hurts anything to add it.